public/views/app.devices.html

```diff
@@ -66,4 +66,176 @@ <h5>{{ device.name }}</h5>
+                      </select>
+                    </div>
+                    <label class="control-label col-sm-4" for="unit">Unit*:</label>
+                    <div class="col-sm-8 fields-space">
+                      <select class="form-control" ng-model="form.dataItem.schema.unit" required>
+                        <option ng-repeat="option in unitValues[form.dataItem.schema.typeID]"
+                        value="{{ option.value }}">{{ option.name }}</option>
+                      </select>
+                    </div>
+                    <label class="control-label col-sm-4" for="valueType">Value Type*:</label>
+                    <div class="col-sm-8 fields-space">
+                      <select class="form-control" ng-model="form.dataItem.schema.valueType" required>
+                        <option value="1" selected>INT</option>
+                        <option value="2">FLOAT</option>
+                        <option value="3">BOOL</option>
+                        <option value="4">RAW</option>
+                        <option value="5">INT64</option>
+                        <option value="6">UINT</option>
+                        <option value="7">UINT64</option>
+                      </select>
+                    </div>
+                  </div>
+                  <div class="col-sm-6">
+                    <div class="col-sm-12 flex-center fields-title">
+                      <h3>Config</h3>
+                    </div>
+                    <label class="control-label col-sm-5" for="lowerThreshold">Lower Threshold:</label>
+                    <div class="col-sm-7 fields-space">
+                      <input type="number" class="form-control" ng-model="form.config.lowerThreshold" placeholder="1000">
+                    </div>
+                    <label class="control-label col-sm-5" for="upperThreshold">Upper Threshold:</label>
+                    <div class="col-sm-7 fields-space">
+                      <input type="number" class="form-control" ng-model="form.config.upperThreshold" placeholder="3000">
+                    </div>
+                    <label class="control-label col-sm-4" for="change">Change:</label>
+                    <div class="col-sm-8 fields-space">
+                      <select class="form-control" ng-model="form.config.change">
+                        <option value="yes" selected>Yes</option>
+                        <option value="No">No</option>
+                      </select>
+                    </div>
+                    <label class="control-label col-sm-4" for="timeSec">Time Sec:</label>
+                    <div class="col-sm-8 fields-space">
+                      <input type="number" class="form-control" ng-model="form.config.timeSec" placeholder="5">
+                    </div>
+                  </div>
+                  <div class="col-sm-12">
+                    <div class="col-sm-12 flex-center fields-title">
+                      <h3>Modbus</h3>
```

Comment:
I think we can have a line above separating the sections in order to improve the visual (there's a lot of information tied together in the model). @Is4bela could help us on this.

Reply:
Done!